src/Illuminate/Foundation/Testing/TestCase.php

```diff
@@ -197,6 +197,11 @@ protected function tearDown(): void
 
             ParallelTesting::callTearDownTestCaseCallbacks($this);
 
+            $database = $this->app['db'];
+            foreach (array_keys($database->getConnections()) as $name) {
+                $database->purge($name);
+            }
```

Comment:
Would be better?

```suggestion
            if ($database = $this->app['db'] ?? null) {
                foreach (array_keys($database->getConnections()) as $name) {
                    $database->purge($name);
                }
            }
```

Reply:
Oops, without realizing this, I fixed it in a different way.
Thank you!